app/javascript/components/UsersList.vue

```diff
@@ -1,38 +1,69 @@
 <template>
   <div>
     <h1>{{ tableName }}</h1>
+      <select v-model="selected" @change='created'>
+        <option value="0">All Projects</option>
+        <option v-for="project in listOfProjects" v-bind:value="project.id" >{{ project.name }}</option>
+      </select>
+      <input type="text" v-model="search" placeholder="Search">
     <table class="table">
       <thead class="thead-dark">
         <tr>
           <th scope="row" v-for="title in theadTitles" >{{ title }}</th>
         </tr>
       </thead>
       <tbody>
-        <tr v-for="(user, index) in listOfUsers">
+        <tr v-for="(user, index) in filteredList">
           <th scope="row">{{ index + 1 }}</th>
           <td>{{ user.first_name }}</td>
           <td>{{ user.last_name }}</td>
           <td>{{ user.email }}</td>
           <td>{{ user.phone }}</td>
-          <td>{{ user.name }}</td>
+          <td>{{ user.project.name }}</td>
         </tr>
       </tbody>
     </table>
   </div>
 </template>
 
 <script>
+
 export default {
   data: function () {
     return {
       listOfUsers: JSON.parse(this.listUsersAll),
       theadTitles: ['#', 'First name', 'Last name', 'Email', 'Phone', 'Project' ],
-      tableName: 'List of users'
+      tableName: 'List of users',
+      listOfProjects: JSON.parse(this.listProjectsAll),
+      selected: '0',
+      search: '',
+      users_url: '/users'
```

Comment:
Why URL is hard coded?  Pass URL to props like `users_path` (Rails route helper methods)